### PR TITLE
IMPUTE2 2.3.0 easyconfigs

### PR DESCRIPTION
**Pull request by pescobar**
this program is distributed only in binary format. No source

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 2 out of 2
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/a5f0bdac165eec2cb19c for a full test report.

File: easybuild/easyconfigs/i/IMPUTE2/IMPUTE2-2.3.0-x86_64_dynamic.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+sources = ['%s_v%s%s.tgz' % (name.lower().rstrip('2'), version, versionsuffix)]
+
+sanity_check_paths = {
+    'files': ["impute2"],
+    'dirs': ["Example"]
+}
+
+# add the top dir to PATH
+modextrapaths = {
+    'PATH': ''
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/i/IMPUTE2/IMPUTE2-2.3.0-x86_64_dynamic.eb]: I believe this should be `'PATH': ['.']`

Comment (pescobar) [inline easybuild/easyconfigs/i/IMPUTE2/IMPUTE2-2.3.0-x86_64_dynamic.eb]: @JensTimmerman I always do it like that and it works.....should I change?

Comment (boegel) [inline easybuild/easyconfigs/i/IMPUTE2/IMPUTE2-2.3.0-x86_64_dynamic.eb]: Hmm, it's weird that `''` works...

Comment (boegel) [inline easybuild/easyconfigs/i/IMPUTE2/IMPUTE2-2.3.0-x86_64_dynamic.eb]: OK, if `eb` notices this isn't a list, it'll make it a list and then continue, see https://github.com/hpcugent/easybuild-framework/blob/master/easybuild/framework/easyblock.py#L881

So, I guess this is fine.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/cde03e511c6ae02a319e for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
filename issue: `IMPUTE2-2.3.0-x86_64_dynamic.eb` vs `IMPUTE2-2.3.0_x86_64_dynamic.eb` (`-` vs `_`)

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**